components/shared/indexIssueDetails/IndexIssueDetails.js

```diff
@@ -135,32 +136,48 @@ function InfoRow(props) {
   );
 }
 
-export default function IndexIssueDetails(props) {
-  const issues = props.issues;
-  const error = props.error;
-  const isLoaded = props.isLoaded;
+export default function IndexIssueDetails({
+  issue_number,
+  issues,
+  indexType,
+  error,
+  isLoaded,
+}) {
   const router = useRouter();
 
   if (error) {
     return <p>{error}</p>;
   } else if (!isLoaded && !error) {
     return <SpinnerWrapper />;
   } else if (issues.length === 0) {
-    return <p>No data found.</p>;
+    // this would suggest we failed to fetch issues. e.g. Socrata is down
+    return (
+      <Alert variant="danger">
+        We're unable to load project data at this time.
+      </Alert>
+    );
+  }
+
+  const issue = issues.filter((issue) => issue.number === issue_number)[0];
+
+  if (!issue) {
+    // this would suggest the user provided an invalid issue ID in the URL
+    return (
+      <Alert variant="danger">
+        No project data found for project ID #{issue.number}
```

Comment:
If the issue is `false`, my take is that you can count on not being able to access the number property of that object, which will throw an error -- very similar to the behavior you described in cityofaustin/atd-data-tech/issues/7298, but emanating from the new error message. 

![image](https://user-images.githubusercontent.com/3653206/135366610-a5c3dcfc-d484-46ee-8071-b2f42a34e0ce.png)

https://deploy-preview-28--atd-product.netlify.app/projects/31337 shows the same error in the deploy preview for this PR.

Reply:
you're completely right—this was a boneheaded mistake on my part 🙏

Reply:
thanks @frankhereford. i just patched a fix by changing the reference to the `issue_number` prop. This value is pulled from the router query in the parent component.

Reply:
<img width="1212" alt="Screen Shot 2021-10-01 at 1 18 08 PM" src="https://user-images.githubusercontent.com/14793120/135668266-69a7f356-607b-4593-876d-94a948e67e6a.png">